### DISCUSSION TITLE
定义中文献引用的问题

### DISCUSSION THREAD

**Discussion by pkeke** [note286/xduts#87] [Q&A] [answered]
请问一下像这样的定义，如何把定义中的这对小括号去掉，谢谢！
![image](https://user-images.githubusercontent.com/44956389/226930859-dd4c86e9-6fb4-4dc4-9e6a-ea7d3db8cbbb.png)

**Comment by note286** [comment]
请给出复现代码。

**Comment by pkeke** [comment]
```
\theoremstyle{definition}
\newtheorem{myDef}{\hspace{2em}定义}[chapter]
\begin{myDef}[\cite{intro}]
	一个DFA是一个五元组$G=(Q,E,\delta,q_0,Q_m)$。其中：
\end{myDef}
```

**Reply by eutopia-61** [reply]
你好！你这边有修改好吗，可以看一下你的设置吗

**Reply by pkeke** [reply]
我没有去掉小括号。

**Reply by note286** [reply]
@eutopia-61 https://github.com/note286/xduts/discussions/105

**Comment by note286** [comment, marked as answer]
请阅读`amsthm`手册，自行修改样式：

![image](https://user-images.githubusercontent.com/16207292/226934188-d13c3381-a25e-4b9a-9762-ed71b7e64bad.png)

**Reply by pkeke** [reply]
好的，谢谢！

**Reply by note286** [reply]
你确定在定义名称的位置就放一个参考文献引用吗？

**Reply by pkeke** [reply]
对的
